packages/broker/src/plugins/operator/MaintainOperatorValueHelper.ts

```diff
@@ -0,0 +1,32 @@
+import { Logger } from '@streamr/utils'
+import { Contract } from 'ethers'
+import { Operator, StreamrConfig, operatorABI, streamrConfigABI } from '@streamr/network-contracts'
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+export class MaintainOperatorValueHelper {
+    config: OperatorServiceConfig
+    operator: Operator
+
+    constructor(config: OperatorServiceConfig) {
+        logger.trace('MaintainOperatorValueHelper created')
+        this.config = config
+        this.operator = new Contract(config.operatorContractAddress, operatorABI, this.config.signer) as unknown as Operator
+    }
+
+    /// returns a wei value (1 ETH means 100%)
+    async getPenaltyLimitFraction(): Promise<bigint> {
+        const streamrConfigAddress = await this.operator.streamrConfig()
+        const streamrConfig = new Contract(streamrConfigAddress, streamrConfigABI, this.config.provider) as unknown as StreamrConfig
+        return (await streamrConfig.poolValueDriftLimitFraction()).toBigInt()
+    }
+
+    async getApproximatePoolValuesPerSponsorship(): Promise<any> {
+        return await this.operator.getApproximatePoolValuesPerSponsorship()
+    }
+
+    async updateApproximatePoolvalueOfSponsorships(sponsorshipAddresses: string[]): Promise<void> {
+        await (await this.operator.updateApproximatePoolvalueOfSponsorships(sponsorshipAddresses)).wait()
```

Comment:
It's a typo. Fixed it for the service method, but the contract one can stay the same for now. Once the ["pull earnings into operator contract"](https://linear.app/streamr/issue/ETH-535/poolvalue-update-mechanism-should-pull-earnings-into-operatorcontract) feature is implemented, it will change name anyways (e.g. does more than just update)